samples/azure-quantum/hello-world/README.md

```diff
@@ -0,0 +1,26 @@
+---
+page_type: sample
+author: alchocro
+description: Hello World Samples
+ms.author: alchocro@microsoft.com
+ms.date: 08/08/2022
+languages:
+- python
+products:
+- azure-quantum
+---
+
+# Hello World Samples
+
+These are Azure Quantum sample notebooks that illustrate how to use Q#, Qiskit, or Cirq to write a simple program and run it against an Azure Quantum provider.
```

Comment:
A bit more summary here would be helpful, since users can discover these samples from the MS-wide samples gallery without necessarily having the context that these are supposed to be quantum notebooks.

Reply:
Added a bit more summary here that it is used in the samples gallery of Azure Quantum workspaces.